sips/minutes/_posts/2016-07-15-sip-minutes.md

```diff
@@ -131,17 +135,17 @@ be resolved before resuming its evaluation.
 revisit the support of async/await in try/catch blocks. When that's considered,
 this SIP should be reopened and and it should see another round of discussion.
 
-#### Proposal SIP-20: Improved lazy val initialization by Dmitry Petrashko
+#### Proposal SIP-20: Improved lazy val initialization presented
 Sebastien reviews the SIP and asks Dmitry, present in the meeting, to correct
 him if he's wrong. He agrees that the SIP is desirable but he's unsure about the
 benchmarks and which of the proposals is faster. Dmitry explains that the
-benchmarks are in the repo, and that in the best case the initialization doesn’t
-need synchronization.
+benchmarks are in the repository.
   
-He also recommends the author to include more documentation.
+Sebastien also recommends the authors to include more documentation.
   
-**Outcome**: Dmitry volunteers to provide an implementation in three months. The
-Process Lead schedules the next iteration by October 2016.
+**Outcome**: Dmitry says he's likely to have time to update the SIP in three
```

Comment:
I would treat it as like the trait parameters. Wait 6 months and see
whether the scalac team has the resources to pick it up for 2.13. They are
certainly interested in having this done.


On Fri, Jul 15, 2016 at 3:05 PM, Dmitry Petrashko <notifications@github.com>
wrote:

> In sips/minutes/_posts/2016-07-15-sip-minutes.md
> <https://github.com/scala/scala.github.com/pull/544#discussion_r70969145>:
>
> >
> > -**Outcome**: Dmitry volunteers to provide an implementation in three months. The
> > -Process Lead schedules the next iteration by October 2016.
> > +**Outcome**: Dmitry says he's likely to have time to update the SIP in three
>
> There are two changes I insist on. Those are:
>
>    - removing my name from the section title. As I'm not the author of
>    the idea. Alex came up with the most complex parts of the idea and praise
>    should go to him;
>    - I didn't promise to provide the implementation for scalac.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scala/scala.github.com/pull/544/files/c426e2f8d6a0b4e1eb587788d0f9841a90f44e8c#r70969145>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAwlVhcI3UeMfH1Fu943ZuLvf9s3GCkPks5qV4WmgaJpZM4JNWA_>
> .
>



-- 

Martin Odersky
EPFL and Lightbend